src/main/java/world/bentobox/challenges/database/object/requirements/StatisticRequirements.java

```diff
@@ -2,227 +2,139 @@
 // Created by BONNe
 // Copyright - 2021
 //
+// Enhanced by tastybento
 
 
 package world.bentobox.challenges.database.object.requirements;
 
 
-import com.google.gson.annotations.Expose;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
 import org.bukkit.Material;
 import org.bukkit.Statistic;
 import org.bukkit.entity.EntityType;
 import org.eclipse.jdt.annotation.Nullable;
 
+import com.google.gson.annotations.Expose;
+
 
+/**
+ * Requirements for statistics based challenges
+ */
 public class StatisticRequirements extends Requirements
 {
     /**
-     * Constructor Requirements creates a new Requirements instance.
+     * Record for this requirement
+     * @param Statistic statistic
+     * @param EntityType entity
+     * @param Material material
+     *  @param Integer amount
+     *  @param Boolean reduceStatistic
      */
-    public StatisticRequirements()
-    {
-        // Empty constructor
+    public record StatisticRec(@Expose Statistic statistic, @Expose EntityType entity, @Expose Material material,
+            @Expose Integer amount, @Expose Boolean reduceStatistic) {
     }
 
-
     /**
-     * This method copies given statistic object.
-     * @return Copy of this object.
+     * Type of the statistic field.
+     * @deprecated Shifting to a list
      */
-    @Override
-    public Requirements copy()
-    {
-        StatisticRequirements requirements = new StatisticRequirements();
-        requirements.setStatistic(this.statistic);
-        requirements.setEntity(this.entity);
-        requirements.setMaterial(this.material);
-        requirements.setAmount(this.amount);
-        requirements.setReduceStatistic(this.reduceStatistic);
-
-        return requirements;
-    }
-
-
-    @Override
-    public boolean isValid()
-    {
-        if (!super.isValid())
-        {
-            return false;
-        }
-
-        if (this.statistic == null)
-        {
-            return false;
-        }
-
-        return switch (this.statistic.getType())
-        {
-            case ITEM -> this.material != null && this.material.isItem();
-            
-            case BLOCK -> this.material != null && this.material.isBlock();
-            
-            case ENTITY -> this.entity != null;
-            
-            case UNTYPED -> true;
-        
-        };
-
-    }
-
-
-    // ---------------------------------------------------------------------
-// Section: Getters and setters
-// ---------------------------------------------------------------------
-
+    @Expose
```

Comment:
This entry is not removed to keep backward compatibility, however, it is never read directly by the code and instead is used when requesting the List of statistics.